core/modules/startup/rootwidget.js

```diff
@@ -45,6 +45,28 @@ exports.startup = function() {
 			}
 		});
 	}
+	// Listen to tag renaming events
+	$tw.rootWidget.addEventListener("tm-rename-tag",function(event) {
+		var params = event.paramObject;
+		if(!params || !params["old"] || !params["new"]) {
+			return;
+		}
+		var filter = "[tag[" + params["old"] + "]!has[draft.of]]";
```

Comment:
Here I'd avoid using a filter. They're a bit brittle (eg not coping with tiddler titles that contain closing square brackets). Instead, you could use `this.wiki.each()` to manually search for the tiddlers you need.